### PR TITLE
[Doc EN]: `guide/configuration.md` review

### PR DESCRIPTION
**Pull request by Haeresis**
In our French translation/review we have changed some consistent things, so we suggest you do the same in the original documentation.

More details in comments.

Do not hesitate to set back the original behavior for some changes if you think they do not make sense.

### CODE REVIEW

File: en/guide/configuration.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
 ---
 title: Configuration
-description: Nuxt.js is configured by default to cover most use cases. This default configuration can be overwritten by using the nuxt.config.js file.
+description: Nuxt.js is configured by default to cover most use cases. This default configuration can be overwritten by using the `nuxt.config.js` file.
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
 ---
 
-> Nuxt.js is configured by default to cover most use cases. This default configuration can be overwritten by using the nuxt.config.js file.
+> Nuxt.js is configured by default to cover most use cases. This default configuration can be overwritten by using the `nuxt.config.js` file.
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
 
 ### build
 
-This option lets you add modules inside the vendor.bundle.js file to reduce the size of the application bundle. This is especially helpful when using external modules.
+This option lets you add modules inside the `vendor.bundle.js` file to reduce the size of the application bundle. This is especially helpful when using external modules.
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
 
-[Documentation about build integration](/api/configuration-build)
+[Documentation about `build` integration](/api/configuration-build)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 5]
```diff
@@ -1,39 +1,39 @@
...
 
 ### cache
 
 This option lets you enable cached components for better render performances.
 
-[Documentation about cache integration](/api/configuration-cache)
+[Documentation about `cache` integration](/api/configuration-cache)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 6]
```diff
@@ -1,39 +1,39 @@
...
 
 ### css
 
 This option lets you define the CSS files/modules/libraries you want to set as globals (included in every page).
 
-[Documentation about css integration](/api/configuration-css)
+[Documentation about `css` integration](/api/configuration-css)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 7]
```diff
@@ -1,39 +1,39 @@
...
 
 ### dev
 
-This option lets you define the development or production mode of nuxt.js
+This option lets you define the `development` or `production` mode of Nuxt.js
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

Comment (Haeresis) [reply en/guide/configuration.md]: Nuxt with capital as a product.

File: en/guide/configuration.md [hunk 8]
```diff
@@ -1,39 +1,39 @@
...
 
-[Documentation about dev integration](/api/configuration-dev)
+[Documentation about `dev` integration](/api/configuration-dev)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 9]
```diff
@@ -1,39 +1,39 @@
...
 
 ### env
 
 This option lets you define environment variables available both the client and the server.
 
-[Documentation about env integration](/api/configuration-env)
+[Documentation about `env` integration](/api/configuration-env)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 10]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
 
 This option lets you to customize the loading component which is loaded by default with Nuxt.js.
 
-[Documentation about loading integration](/api/configuration-loading)
+[Documentation about `loading` integration](/api/configuration-loading)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 11]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
 
 ### modules
 
-This option lets you add nuxt modules to your project.
+This option lets you add Nuxt modules to your project.
```
Comment (Haeresis) [inline en/guide/configuration.md]: Nuxt with capital as a no-.js Product Name

File: en/guide/configuration.md [hunk 12]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
 
-[Documentation about modules integration](/api/configuration-modules)
+[Documentation about `modules` integration](/api/configuration-modules)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 13]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
 
 ### plugins
 
-This option lets you to define JavaScript plugins to be run before instantiating the root vue.js application.
+This option lets you to define JavaScript plugins to be run before instantiating the root Vue.js Application.
```
Comment (Haeresis) [inline en/guide/configuration.md]: More consistency with « Universal Vue.js Applications »

File: en/guide/configuration.md [hunk 14]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
 
-[Documentation about plugins integration](/api/configuration-plugins)
+[Documentation about `plugins` integration](/api/configuration-plugins)
 
 ### rootDir
 
-This option lets you define the workspace of your nuxt.js application.
+This option lets you define the workspace of your Nuxt.js Application.
```
Comment (Haeresis) [inline en/guide/configuration.md]: More consistency with previous « Vue.js Application »

File: en/guide/configuration.md [hunk 15]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
 
-[Documentation about rootDir integration](/api/configuration-rootdir)
+[Documentation about `rootDir` integration](/api/configuration-rootdir)
 
 ### router
 
-This option lets you to overwrite the default Nuxt.js configuration of vue-router.
+This option lets you to overwrite the default Nuxt.js configuration of Vue Router.
```
Comment (Haeresis) [inline en/guide/configuration.md]: Use the product name for Vue Router (not package name) for more conistency with Nuxt.js.

File: en/guide/configuration.md [hunk 16]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
 
-[Documentation about router integration](/api/configuration-router)
+[Documentation about `router` integration](/api/configuration-router)
 
 ### srcDir
 
-This option lets you define the source directory of your nuxt.js application.
+This option lets you define the source directory of your Nuxt.js Application.
```
Comment (Haeresis) [inline en/guide/configuration.md]: Same as above.

File: en/guide/configuration.md [hunk 17]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
-[Documentation about rootDir integration](/api/configuration-rootdir)
+[Documentation about `rootDir` integration](/api/configuration-rootdir)
 
 ### router
 
-This option lets you to overwrite the default Nuxt.js configuration of vue-router.
+This option lets you to overwrite the default Nuxt.js configuration of Vue Router.
 
-[Documentation about router integration](/api/configuration-router)
+[Documentation about `router` integration](/api/configuration-router)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 18]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
-[Documentation about plugins integration](/api/configuration-plugins)
+[Documentation about `plugins` integration](/api/configuration-plugins)
 
 ### rootDir
 
-This option lets you define the workspace of your nuxt.js application.
+This option lets you define the workspace of your Nuxt.js Application.
 
-[Documentation about rootDir integration](/api/configuration-rootdir)
+[Documentation about `rootDir` integration](/api/configuration-rootdir)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 19]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
-[Documentation about modules integration](/api/configuration-modules)
+[Documentation about `modules` integration](/api/configuration-modules)
 
 ### plugins
 
-This option lets you to define JavaScript plugins to be run before instantiating the root vue.js application.
+This option lets you to define JavaScript plugins to be run before instantiating the root Vue.js Application.
 
-[Documentation about plugins integration](/api/configuration-plugins)
+[Documentation about `plugins` integration](/api/configuration-plugins)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 20]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
-[Documentation about router integration](/api/configuration-router)
+[Documentation about `router` integration](/api/configuration-router)
 
 ### srcDir
 
-This option lets you define the source directory of your nuxt.js application.
+This option lets you define the source directory of your Nuxt.js Application.
 
-[Documentation about srcDir integration](/api/configuration-srcdir)
+[Documentation about `srcDir` integration](/api/configuration-srcdir)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

File: en/guide/configuration.md [hunk 21]
```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
...
 
 ### transition
 
 This option lets you define the default properties of the page transitions.
 
-[Documentation about transition integration](/api/configuration-transition)
+[Documentation about `transition` integration](/api/configuration-transition)
```
Comment (Haeresis) [inline en/guide/configuration.md]: To separate content from code / path / filename / command / etc.

**Comment by Haeresis** [comment]
Hi, @Atinux, today I suggest you this first PR on five.

**Comment by Atinux** [comment]
Excellent job @Haeresis :raised_hands:

**Pull request merged**